apps/files/lib/cloudcmd/lib/cloudfunc.js

```diff
@@ -255,7 +256,7 @@
 
                 // OSC_CUSTOM_CODE call buildPrettyDate() function and use blank date instead of '--.--.----'
                 date    = buildPrettyDate(file.date) || '          ';
-                owner   = file.owner || 'root';
+                owner = execSync('id -nu ' + file.owner) || 'root';
```

Comment:
If you are going to use execSync you need to handle exceptions in a try catch block. So I'd keep the default `owner   = file.owner || 'root';` and then after that add a pseudocode `try{ if(file.owner){ owner = execSync('id -nu ' + file.owner) }}catch...` block.